DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyBase.cs

```diff
@@ -1624,6 +1624,12 @@ protected void ClearReadyStatuses()
             }
         }
 
+        protected void UpdateUIPlayerNameWithPing(PlayerInfo pInfo)
+        {
+            XNADropDown ddPlayerName = ddPlayerNames[pInfo.Index];
+            ddPlayerName.Items[0].Text = $"{pInfo.Name} " + (pInfo.Ping >= 0 ? $"{pInfo.Ping.ToString()} ms" : "? ms");
```

Comment:
Would be great to have those as tooltips on work-in-progress ping indicator.

Reply:
Agreed, I can definitely implement that

Reply:
@Metadorius I'm thinking I'll include the tooltips enhancement in a separate PR since the main value added will be from the icons.

Reply:
In process of implementing them.